src/npc_manager.py

```diff
@@ -62,7 +62,7 @@ def open_npc_menu(self, npc_key: Npc) -> bool:
             img = self._screen.grab()
             results = []
             for key in self._npcs[npc_key]["template_group"]:
-                res, pos = self._template_finder.search(key, img, threshold=0.35, roi=roi)
+                res, pos = self._template_finder.search(key, img, threshold=0.35, roi=roi, normalize_monitor=True)
```

Comment:
if you normalize to monitor here you will have to remove it in line 72. Otherwise you will do it twice.